easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -722,6 +722,26 @@ def get(self, key, default=None):
         else:
             return default
 
```

Comment:
`__ne__` should also be implemented, see https://github.com/ocaisa/easybuild-framework/pull/17